trade-finance-manager-ui/server/controllers/helpers/can-deal-be-cancelled.helper.ts

```diff
@@ -16,14 +23,19 @@ export const canSubmissionTypeBeCancelled = (submissionType: DealSubmissionType)
  * Checks if deal cancellation is enabled for a deal and user type
  * @param submissionType - the deal submission type
  * @param user - the session user
+ * @param cancellationStatus - the current deal cancellation status if it exists
  * @returns true or false depending on the feature flag, submission type and user type
  */
-export const isDealCancellationEnabled = (submissionType: DealSubmissionType, user: TfmSessionUser): boolean => {
-  // TODO: DTFS2-7298: also check that the deal hasn't already been cancelled.
+export const canDealBeCancelled = (submissionType: DealSubmissionType, user: TfmSessionUser, cancellationStatus?: TfmDealCancellationStatus): boolean => {
```

Comment:
I've renamed this as I think the new name is a tad clearer (makes it clear it's not just based on the feature flag being enabled)